lib/specs/latest.js

```diff
@@ -16,6 +16,14 @@ let rules = {};
 // assign new or overwrite existing knownHelpers, templates, or rules here:
 rules = {
     // New rules
+    'GS001-DEPR-USER-GET': {
+        level: 'error',
+        rule: `<code>{{#get "users"}}</code> should be replaced with <code>{{#get "authors"}}</code>`,
+        details: `The usage of <code>{{#get "users"}}</code> is deprecated and should be replaced with <code>{{#get "authors"}}</code>.<br>
+        Find more information about the <code>{{get}}</code> helper <a href="${docsBaseUrl}get" target=_blank>here</a>.`,
+        regex: /{{\s*?#get ("|')\s*users("|')\s*/g,
+        helper: '{{#get "users"}}'
```

Comment:
This needs to also cope with people who use single quotes, and also handle other whitespace patterns.